go.mod

```diff
@@ -5,26 +5,23 @@ go 1.16
 require (
 	github.com/CosmWasm/wasmd v1.0.0-beta5
 	github.com/CosmWasm/wasmvm v1.0.0-beta5
```

Comment:
wasmd `v0.25.0` updates wasmvm to v1.0.0-beta10 (reference in the fork go.mod) we should also update it in here (I'd put a code suggestion but we also need to update the go.sum which can't be done in code suggestion)